src/quickpicks/common.ts

```diff
@@ -122,18 +124,30 @@ export async function selectAndroidKeystore (lastUsed) {
 			id: 'last'
 		});
 	}
+	if (savedKeystorePath) {
+		items.push({
+			label: `Saved ${savedKeystorePath}`,
+			id: 'saved'
+		});
+	}
 	const keystoreAction = await quickPick(items, { placeHolder: 'Browse for keystore or use last keystore' });
 	if (keystoreAction.id === 'browse') {
 		const uri = await window.showOpenDialog({ canSelectFolders: false, canSelectMany: false });
 		return uri[0].path;
+	} else if (keystoreAction.id === 'saved') {
+			return savedKeystorePath;
 	} else {
 		return lastUsed;
 	}
 }
 
-export async function enterAndroidKeystoreInfo (lastUsed) {
-	const location = await selectAndroidKeystore(lastUsed);
-	const alias = await inputBox({ placeHolder: 'Enter your keystore alias' });
+export async function enterAndroidKeystoreInfo (lastUsed, savedKeystorePath) {
+	const location = await selectAndroidKeystore(lastUsed, savedKeystorePath);
+
+	if (!await pathExists(location)) {
+		throw new InteractionError('The Keystore files does not exist');
```

Comment:
I think this should be singular `file`, might be good to provide the provided location too to give some context

```suggestion
		throw new InteractionError(`The Keystore file ${location} does not exist`);
```